### ISSUE TITLE
Nothing happened in GAZEBO

### ISSUE THREAD

**Issue by zychaoqun** [AlessioTonioni/Autonomous-Flight-ROS#6]
I have compiled the package successfully.It seems that when I typed: sh 1-simulator.sh and 2-controller.sh and 3-planner sh, I can see the GUI successfully. and when I run sh 3-planner, I can see the gui of rviz. While firstly , there is no fixed frame called map,when I change the frame to base_link ,and add the moveit plugin, I can see the quadrotor with iteratct markers. When I select a end point for the quadrotor and press "plan and execute", the quadrotor can fly to its destination successfully. But nothings happens in Gazebo, the quadrotor in gazebo just land without any movement. What should I do next if I want to see the octomap built by the quadtor? Or is there anything go wrong as I described above? So many thanks!

**Comment by AlessioTonioni** [comment]
If I understood well you can sucesfully plan a trajectory but it is not executed from the simulated quadrotor. The component responsible for the execution of the trajectory is the node spawned by 'controller.sh', it receives messages from moveit and translate it in 'cmd_vel' that the quadrotor can execute. Check the output of the sh to see if there are some strange error messages.

If you can't see the cotomap in rviz maybe gazebo is not working at all or is not publishing any kind of message from the sensors, are you able to display some of the simuated sensors? e.g. camera, kinect or imu?

**Comment by zychaoqun** [comment]
Thanks for your reply!
I can received the topics published by the sensors,e.g. camera ,kinect ,etc.
And I can see the gazebo is receiving the /md_vel topic from action_controller node, while when I use rostopic echo to see what is in /cmd_vel , it just says, is the /clock running,I have check the topic /clock, it seems working well. And there is no other message of /cmd_del.

should I do something after launching the 1-3.sh files? I can use the plan and execute in rviz but can not make the quadrotor in gazebo fly and get the octomap.

Thanks!

**Comment by zychaoqun** [comment]
Also ,After I launched gazebo and rqt, I found that gazebo can receive the /cmd_vel topic, but there is no such topic  "move_status" and "command/motor" as other quadrotor simulations do. Maybe that is why the quadrotor can't move?

**Comment by AlessioTonioni** [comment]
yes that could be the problem, check if everything is correctly spawned from the quadrotore.urdf.

Are you able to control the drone using teleOp.sh?

**Comment by zychaoqun** [comment]
Yes I can't control the quadrotor using the teleOp.sh, I opened rviz and there is no such kind of topic "command/motor" connect gazebo. Could you please tell me how to check whether everything is spawned from the quadrotore?
